### ISSUE TITLE
Long camera names throws uninformative errors 

### ISSUE THREAD

**Issue by vikashplus** [deepmind/mujoco#963]
If the camera name is beyond certain size, loading the model throws an uninformative errors that's hard to parse 
```xml
<mujoco model="table-environment">
    <worldbody>
        <body name="mmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmm">
            <camera name="wwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwww"/> <!-- A long camera name throws an error even though its shorter than other names-->
            <geom name="ccccccccccccccccccccccccccccccccccccccccccccccccccc" type="sphere" size="1"/>
        </body>
    </worldbody>
</mujoco>
```

error 
```
MuJoCo version 2.3.6
Plugins registered by library 'libelasticity.so':
    mujoco.elasticity.cable
    mujoco.elasticity.solid
Plugins registered by library 'libsensor.so':
    mujoco.sensor.touch_grid
ERROR: mjui_add: invalid multi element name

Press Enter to exit ...
```

- Include the following context:
  - Operating system: Ubuntu 18.08
  - MuJoCo version: 2.3.6

**Comment by yuvaltassa** [comment]
Ah yes I remember this, this is due to the camera selection menu using camera names (in the render tab), so the UI errors (names are unbounded but menu items are). Very easy to fix (concatenate the name string sent to the UI). Feel free to send a PR, if not I will get to it soon.
